Components/FlightControl/RocketSM.cpp

```diff
@@ -361,6 +361,7 @@ RocketState Arm::OnEnter()
     // We don't do anything upon entering arm
     // TODO: Consider automatically beginning arm sequence (since we've already explicitly entered the arm state)
 	GPIO::MEV_EN::On();
+	GPIO::PowerSelect::InternalPower();
```

Comment:
Note (for procedure) that we should probably be transitioning power _before_ Arm as well (manually using the command)